policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/basic_personal_amount/base_amount.yaml

```diff
@@ -0,0 +1,14 @@
+description: Yukon credits this base basic personal amount.
+values:
+  2022-01-01: 12_719
+  2023-01-01: 13_521
+metadata:
+  unit: currency-CAD
+  label: Yukon income basic personal base amount
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf#page=1
+  - title: Worksheet for the Yukon 2022 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-22e.pdf#page=1
+  # The credit amount is not mentioned in the Yukon income tax act
```

Comment:
it should be there "amount for self" or similar

Reply:
what does that mean? I am a bit confused.